website/docs/docs/advanced-usage/components.md

```diff
@@ -200,6 +201,15 @@ For situations when you strictly want to show some code and run it in the browse
   </SandpackLayout>
 </SandpackProvider>
 
+### Decorators
+
+Decorators provide a way to draw or style a piece of the editor, and you can implement it in the following ways:
```

Comment:
That line confuses me. For decorators I think of JS decorators, but not sure this is the same thing.

Reply:
Great suggestion, yeah that makes sense to me!